examples/ct/ct_handlers.go

```diff
@@ -19,21 +30,28 @@ const (
 )
 
 const (
-	jsonMapKeyCertificates string = "certificates"
+	contentTypeHeader      string     = "Content-Type"
+	contentTypeJSON        string     = "application/json"
+	jsonMapKeyCertificates string     = "certificates"
+	logVerboseLevel        glog.Level = 2
 )
 
 // CTRequestHandlers provides HTTP handler functions for CT V1 as defined in RFC 6962
 // and functionality to translate CT client requests into forms that can be served by a
 // log backend RPC service.
 type CTRequestHandlers struct {
```

Comment:
It was the CTRequestHandlers members I wanted documented, not the constants above.